pom.xml

```diff
@@ -301,6 +305,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <phase>package</phase>
```

Comment:
_nit_ The default phase of the `jar` goal is `package`, so the phase does not need to be configured here.